UnityProject/Assets/Scripts/Objects/Doors/FalseWallTileBlend.cs

```diff
@@ -0,0 +1,245 @@
+using Doors;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Tiles;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+
+namespace Objects.Doors
+{
+    public class FalseWallTileBlend : MonoBehaviour
+    {
+        [HideInInspector] private SpriteHandler SPRITE_HANDLER;
```

Comment:
```suggestion
        [HideInInspector] private SpriteHandler spriteHandler;
```

we use camelCase to name private fields. please rename all your private fields to follow this convention.

Reply:
Ok